src/nreplController.ts

```diff
@@ -46,13 +46,19 @@ const start = (): Promise<CljConnectionInformation> => {
         });
 
         nreplProcess.stderr.on('data', data => {
+            console.info('nrepl stderr =>', data.toString());
+        });
+
+        nreplProcess.on('exit', (code, signal) => {
+            console.info(`nREPL exit => ${code} / Signal: ${signal}`);
             stop();
-            return reject(`This error happened with our nREPL process: ${data}`);
+            return reject();
```

Comment:
Just checking: Do we want to reject() even if the exit code is 0? I suppose we do, if it exits before we manage to resolve() the Promise by getting the connection info, we should probably treat that as an error. What do you think?